tecken/api/forms.py

```diff
@@ -64,24 +64,6 @@ def clean_state(self):
         return value
 
 
-class UserEditForm(forms.ModelForm):
```

Comment:
I dropped the web ui user admin in favor of the Django admin user pages to reduce the maintenance burden.